modules/post/windows/manage/ducky.rb

```diff
@@ -0,0 +1,137 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Windows Ducky Script Parser',
+        'Description' => %q{
+          This module can be used to execute supplied *lightweight* Ducky Scripts - only supports: GUI, STRING, and STRINGLN.
+          It also supports sending individual keyevents based on Metasploit's current standards.
+
+          Setting VERBOSE to true will output each line of your file as it is parsed.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'dru1d <tyler.booth[at]cdw.com>',
+        ],
+        'References' => [
+          'URL', 'https://docs.hak5.org/hak5-usb-rubber-ducky/duckyscript-tm-quick-reference',
+        ],
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [],
+          'SideEffects' => [CONFIG_CHANGES, ARTIFACTS_ON_DISK]
```

Comment:
```suggestion
          'SideEffects' => [CONFIG_CHANGES, ARTIFACTS_ON_DISK, SCREEN_EFFECTS]
```

And _maybe_ these ones too?

https://github.com/rapid7/metasploit-framework/blob/07a91df7a12b7629c28a14a92a21ff9778b117e9/lib/msf/core/constants.rb#L87-L90

Reply:
I am open to whatever is accepted by the dev team. It definitely would create movement on the device and has the potential to also trigger audio. I didn't want to overdo it and put a laundry list of attributes in my first commit. :)